code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -626,15 +631,18 @@ pub mod pallet {
 					owner,
 					asset_id: pool_asset,
 					reward_configs: initial_reward_config,
+					start_block,
 					end_block,
 					lock,
 					share_asset_id,
 					financial_nft_asset_id,
 				} => {
+					// now <= start_block < end_block
 					ensure!(
-						end_block > frame_system::Pallet::<T>::current_block_number(),
-						Error::<T>::EndBlockMustBeInTheFuture
+						start_block >= frame_system::Pallet::<T>::current_block_number(),
```

Comment:
```suggestion
						start_block > frame_system::Pallet::<T>::current_block_number(),
```
I think it's best to allow it to be from next block to prevent any errors/attack that could be caused by initialization not being final. The current block can contain transactions for this new pool that take advantage of this otherwise

Reply:
c65b1e665cf3712c7cdede65c1cd5b9a926c46e3